### PR TITLE
Add proprietarily-licensed font files

### PR DESCRIPTION
**Pull request by willbarton**
This changes adds our proprietary font files to this repository, with a note in the README regarding their licensing.

This change also removes the "ci" frontend build target which was only used for faking these font files in public CI.

## Checklist

- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -336,7 +326,7 @@ that are specific to Postgres. The `CREATEDB` keyword above allows the
 After you have chosen a means to run PostgreSQL and Elasticsearch,
 [set up the environment](#set-up-the-environment),
 [set up a local Python environment](#set-up-a-local-python-environment),
-optionally [installed our private fonts](#install-our-private-fonts),
+optionally [used our private fonts from a CDN](#use-our-private-fonts-from-a-cdn),
```
[the remark this answers is not in the document]
Comment (willbarton) [reply docs/installation.md:329]: The rest of the sentence is past-tense. It's an awkward sentence though. I think I'll rewrite it entirely.

**Comment by anselmbradford** [comment]
I wonder if we should just add the fonts we use. I think we put them all in the private repo for reference if needed, but doesn't seem like we need them all here (and can always grab them from the private repo, if needed). The files we use are these two for regular: https://github.com/cfpb/design-system/blob/main/packages/cfpb-typography/src/licensed-fonts.less#L52-L53

And these two for medium: https://github.com/cfpb/design-system/blob/main/packages/cfpb-typography/src/licensed-fonts.less#L62-L63

**Comment by wpears** [comment]
>I wonder if we should just add the fonts we use. 

I endorse this!

**Comment by willbarton** [comment]
@anselmbradford @wpears alrighty, I'll remove everything that's not those files!

File: docs/installation.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (willbarton) [reply docs/installation.md]: Fixed in [d16e655](https://github.com/cfpb/consumerfinance.gov/pull/7025/commits/d16e655d3077ddb682e68a738f5ab6c43cc9eeaf)

**Review by wpears — APPROVED** [review]

**Pull request merged**